There is no -Isrc/headers, thus specify gpm.h relative location

--- a/src/prog/get-versions.c	2012-10-26 23:21:38.000000000 +0200
+++ b/src/prog/get-versions.c	2015-06-22 15:49:30.536004939 +0200
@@ -22,7 +22,7 @@
  ********/
 
 #include <stdio.h>            /* printf()             */
-#include <gpm.h>              /* gpm information      */
+#include "headers/gpm.h"      /* gpm information      */
 
 int main()
 {
